Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Start from a rust base image
+FROM rust:1.56
+
+# Set the current directory
+WORKDIR /app
+
+# Copy everthing that is not dockerignored to the image
+COPY . .
+
+# Install
+RUN apt-get -y update
+RUN apt-get -y upgrade
+RUN apt-get install -y nodejs npm
+RUN npm install --global yarn
+RUN make install
+
+# Prepare
+RUN rustup toolchain install nightly-2021-07-29-x86_64
```

Comment:
That was the thing I had troubles yesterday. For me it did not work out to install just "nightly". The `component add` step needed exactly this toolchain to be installed.

Reply:
Acatually, this is correct since we need to compile the WASM code for the webworkers which relies on this specific toolchain. Sepcifically, wasm-bindgen-rayon requires it.